src/workloads/execution/AggregateExpressions.yml

```diff
@@ -0,0 +1,661 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-execution"
+Description: |
+  This workload measures aggregation expression performance.
+
+  While measuring performances, this workload collects numbers
+  for either the SBE or the classic engine aggregation expression
+  implementations, depending on environments that it runs on.
+
+  Numbers on the 'standalone-all-feature-flags' environment are for
+  the SBE aggregation expressions and numbers on the 'standalone' environment
+  for the classic aggregation expressions.
+
+Keywords:
+- aggregate
+- sbe
+
+GlobalDefaults:
+  Database: &Database test
+  Collection: &Collection Collection0
+  DocumentCount: &DocumentCount 1e2
```

Comment:
I would say 10 documents is a bit small for a "realistic collection", how about 10k (100k might be too much though; it is up to you)?